index.js

```diff
@@ -2312,11 +2312,41 @@ function measureText(font, text) {
 
 /**
  * Writes the image to a file
- * @param path a path to the destination file (either PNG or JPEG)
- * @param (optional) cb a function to call when the image is saved to disk
- * @returns this for chaining of methods
+ *
+ * @param {string} path a path to the destination file (either PNG or JPEG)
+ * @param {null|function} cb a function to call when the image is saved to disk
+ * @param {boolean} returnPromise
+ * @returns {Jimp|Promise} if returnPromise is true then returns a Promise otherwise `this` for chaining methods
+ */
+Jimp.prototype.write = function (path, cb, returnPromise) {
```

Comment:
It will be BC break. @oliver-moran Can you please decide as repo author? Thanks!